src/server/scripts/Commands/cs_character.cpp

```diff
@@ -713,22 +712,22 @@ class character_commandscript : public CommandScript
         return true;
     }
 
-    static bool HandleCharacterEraseCommand(ChatHandler* handler, PlayerIdentifier player)
+    static bool HandleCharacterEraseCommand(ChatHandler* handler, PlayerIdentifier playerIdent)
```

Comment:
I don't like this, other commands use `PlayerIdentifier player` and `Player* target`.

Reply:
like i'm not saying it needs to stay like that but try to keep them unified i guess?